packages/experimental/Text/src/Text.types.ts

```diff
@@ -0,0 +1,112 @@
+import {
+  FontTokens,
+  FontVariantTokens,
+  buildUseTokens
+} from '@fluentui-react-native/framework';
+import { ITextProps } from '@fluentui-react-native/adapters';
+import { ColorValue } from 'react-native';
+
+export const textName = 'Text';
+
+/**
+ * Text tokens, these are the internally configurable values for Text elements. In particular these
+ * drive decisions on how to build the styles
+ */
+ export interface TextTokens extends FontTokens {
+  /** foreground text color */
+  color?: ColorValue;
```

Comment:
I think IForegroundColorTokens would cover the color prop, so you could have TextTokens extend IForegroundCOlorTokens as well.